resume.yaml

```diff
@@ -1,121 +0,0 @@
-name: Sean Gatewood
-contact:
-  email: stg2bd@virginia.edu
-  linkedin: linkedin.com/in/sean-t-gatewood
-  github: github.com/sgatewood
-education:
-  school: University of Virginia
-  school titles:
-    - Rodman Scholar
-    - Raven Society
-  degree: B.S. Computer Science
-  gpa: 3.95
-  graduation: May 2020
-  notable courses:
-    - Computer Graphics
-    - Algorithms
-    - Advanced Software Development
-    - Introduction to Cybersecurity
-    - Software Testing
-experience:
-  - title: Software Engineer II
-    company: Appian Corporation
-    location: McLean, VA
-    dates: August 2020 - present
-    notes:
-      - Currently work on a SCRUM team that creates and maintains a variety of internal developer tooling and infrastructure.
-      - Helped design and implement a modern feature toggles framework for a legacy codebase. This project was pretty extensive and incorporated a variety of tooling and services, including the creation of standalone services with Kubernetes. 
-      - Performed numerous spikes, some of which turned into real quarter-long features.
-      - Regularly initiated & participated in architectural design discussions.
-      - Treasured high code quality standards since our team's culture was to set good examples for the department.
-  - title: Software Engineering Intern
-    company: Uber Advanced Technologies Group
-    location: Pittsburgh, PA
-    dates: May 2019 - August 2019
-    notes:
-      - Developed implementations for a low-level key-value store using various database technologies (e.g. LevelDB, Sqlite3)
-      - Performed an experiment using Google Benchmark to measure the relative performances of implementations.
-      - Gained experience with real-world workflows and development systems.
-      - Presented results to Uber ATG software engineers.
-  - title: Teaching Assistant - Introduction to Programming
-    company: University of Virginia
-    location: Charlottesville, VA
-    dates: Fall 2017 - May 2020
-    notes:
-      - Assisted hundreds of students with introductory programming concepts in Python.
-      - Created <a href="https://www.youtube.com/playlist?list=PLeih3T8PoRaYXpRwCVUmCEQqzZ51qyAmD">review videos</a> and study material for the students.
-      - Helped manage the course as one of four "Head TAs."
-  - title: Software Engineering Intern
-    company: NASA Langley Research Center
-    location: Hampton, VA
-    dates: May 2018 - May 2019
-    notes:
-      - "Data Management for Airborne Atmospheric Composition Data."
-      - Added HDF file compatibility to NASA’s <a href="https://www-air.larc.nasa.gov/missions/TOLNet/tools/FTScan.zip">TOLNet File Format Scanner and Curtain/Profiles Plotter</a>.
-      - Engineered a <a href="https://www-air.larc.nasa.gov/cgi-bin/Driver.cgi?platform=KORUSAQ/DC8_AIRCRAFT">full-stack web application</a> (JQuery, AJAX, cgi, C++) for atmospheric researchers to create customized data merges.
```

Comment:
Whoa, nice. I've never seen this before! This line is really good in terms of voice and information presentation. Try to make most of your lines read like this if you can. 

The PDF generator or CSS is doing something funky with the spacing or link rendering or something
<img width="398" alt="image" src="https://user-images.githubusercontent.com/16855664/207466443-47c377e3-456b-4c9c-a8e9-ba938e136550.png">